src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGenerator.cs

```diff
@@ -11,6 +12,8 @@ public class XamlCodeGenerator : SourceGenerator
 	{
 		public override void Execute(SourceGeneratorContext context)
 		{
+			// Debugger.Launch();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6166183)